tests/framework/matcher/status.go

```diff
@@ -82,3 +84,11 @@ func HaveStateChangeRequests() gomegatypes.GomegaMatcher {
 		}),
 	}))
 }
+
+func FirstInterfaceIPsConsistOf(expectedAddresses []string) gomegatypes.GomegaMatcher {
```

Comment:
why do you assume the first interface ? How will this ever work with multiple interfaces ? 

I would at least replace "first" with "only" and in a docstring this function is only to be used on VMI's with a single interface. 

... or, make this work for multiple interfaces, for which the client would have to indicate which of the interfaces they are interested on.

Reply:
Refactored it to find the interface by name, so it's more useful for future users.